src/main/java/com/cst438/controller/EnrollmentController.java

```diff
@@ -18,27 +17,35 @@
 @CrossOrigin(origins = "http://localhost:3000")
 public class EnrollmentController {
 
-    // instructor downloads student enrollments for a section, ordered by student name
+    @Autowired
+    EnrollmentRepository enrollmentRepository;
+
+    // instructor downloads student enrollments for a section, ordered by student
+    // name
     // user must be instructor for the section
     @GetMapping("/sections/{sectionNo}/enrollments")
     public List<EnrollmentDTO> getEnrollments(
-            @PathVariable("sectionNo") int sectionNo,
-            EnrollmentRepository enrollmentRepository) {
+            @PathVariable("sectionNo") int sectionNo) {
 
         // TODO
-        //  hint: use enrollment repository findEnrollmentsBySectionNoOrderByStudentName method
+        // hint: use enrollment repository findEnrollmentsBySectionNoOrderByStudentName
+        // method
 
         // Fetch enrollments from repository
         List<Enrollment> enrollments = enrollmentRepository.findEnrollmentsBySectionNoOrderByStudentName(sectionNo);
 
+        if (enrollments.size() == 0) {
```

Comment:
Alternative: `enrollments.isEmpty()`

Reply:
Should an empty list be an error state? Perhaps the `sectionNo` could be validated separately, and an empty list returned for a section with no enrollments.

Reply:
Both suggestions implemented.